lab_1/main.py

```diff
@@ -40,7 +65,20 @@ def get_top_n_words(freq_dict: dict, top_n: int) -> list or None:
     :param top_n: a number of the most common words
     :return: a list of the most common words
     """
-    pass
+    if not isinstance(freq_dict, dict):
+        return None
+    if not isinstance(top_n, int):
+        return None
+    inf = list(freq_dict.items())
+    inf = sorted(inf, key=lambda x: x[1], reverse=True)
```

Comment:
sort by keys, not items

Reply:
Sorry, I don't really understand... all items are sorted by values which are the frequency of words.

Reply:
yes, and I tell you to sort keys(),` freq_dict.items()` -> `freq_dict.keys()` or better `freq_dict`
This way you don't need extra `for` loop